test/js/ui/camera.test.js

```diff
@@ -979,21 +979,21 @@ test('camera', (t) => {
         t.test('peaks at the specified zoom level', (t) => {
             const camera = createCamera();
             camera.setZoom(20);
-            let minZoom = Infinity;
+
+            const minZoom = 1;
 
             camera.on('zoom', () => {
-                if (camera.getZoom() < minZoom) {
-                    minZoom = camera.getZoom();
+                const zoom = camera.getZoom();
+                if (zoom < 1) {
```

Comment:
I like this, but we still need a check to ensure zoom has changed, which is integral to the `fly` animation. How about we increase the leniency of the `t.ok(fixedNum(minZoom, 2) < 1.1)` to something like `t.ok(fixedNum(minZoom, 2) < 2);`? The zoom event fires regularly enough that this won't be as flaky.